api/src/test/java/gov/cms/ab2d/api/service/JobServiceTest.java

```diff
@@ -48,4 +50,21 @@ public void createRequest() {
     public void failedValidation() {
         jobService.createJob("Patient,ExplanationOfBenefits,Coverage", "http://localhost:8080");
     }
+
+    @Test
+    public void cancelJob() {
+        Job job = jobService.createJob("ExplanationOfBenefits", "http://localhost:8080");
+
+        jobService.cancelJob(job.getJobID());
+
+        // Verify that it has the correct status
+        Job cancelledJob = jobRepository.findByJobID(job.getJobID());
+
+        assertEquals(JobStatus.CANCELLED, cancelledJob.getStatus());
+    }
+
+    @Test(expected = EntityNotFoundException.class)
```

Comment:
Similarly, will need test cases to cover attempts to cancel FAILED or COMPLETED job (see previous comments)